core/lib/types/src/api/mod.rs

```diff
@@ -733,3 +735,76 @@ pub struct Proof {
     pub address: Address,
     pub storage_proof: Vec<StorageProof>,
 }
+
+/// Collection of overridden accounts, useful for `eth_estimateGas`.
+#[derive(Debug, Clone, Serialize, Deserialize)]
+pub struct StateOverride(pub HashMap<Address, OverrideAccount>);
```

Comment:
do we need pub here?

Reply:
Fixed in 0327102d39f3b5d5177973e1a730c92eee962bdb